hibike/devices/BatteryBuzzer/pindefs.h

```diff
@@ -0,0 +1,26 @@
+#define R2 10.0
```

Comment:
So these definitions are not pindefs, as the name of the file implies.

Also it might be good to have a comment, that among other things says that these resistor values are measured in KOhm (?)